src/convertToObject.js

```diff
@@ -6,7 +6,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceArray = sourceString.split(';');
```

Comment:
[CODE KNOWLEDGE] - There's no need to make a copy before each operation. The `split` method doesn't mutate the original string, so you can directly use it without assigning it to `sourceArray`.